src/features/Main/components/CardPartner/index.tsx

```diff
@@ -8,18 +8,30 @@ import {
 import { FC } from 'react';
 
 const CardPartners: FC = () => {
+  const Images = [
+    { src: 'efishery.png' },
+    { src: 'icehouse.png' },
+    { src: 'learning.png' },
+    { src: 'shipper.png' }
+  ];
```

Comment:
set to constants folder, and create one file like images.ts on the constants, paste the array @tjoetmuty